### ISSUE TITLE
Computed properties break _rerender

### ISSUE THREAD

**Issue by aphitiel** [ekhaled/svelte-dev-helper#6]
When a component uses computed properties, `_rerender` tries to `set()` those, which makes `_checkReadOnly` inside the component throw. This leads to a full reload when using svelte-loader with hotReload.

I couldn't find way to tell if a property is computed, though a possible workaround seems to be replacing [this line](https://github.com/ekhaled/svelte-dev-helper/blob/560b85517ac737a0011ff5fee29f15ee34c5abd7/lib/proxy.js#L201) with
```
  this.proxyTarget._updatingReadonlyProperty = true;
  this.set(oldstate);
  this.proxyTarget._updatingReadonlyProperty = false;
```

Although that feels somewhat dirty. What do you think?

**Comment by ekhaled** [comment]
cc. @Rich-Harris
I don't know if I remember right, but I seem to remember a fix in svelte where trying to set a computed prop would result in a no-op. 

Maybe Rich can enlighten us... I'm on mobile now, but I'll try to find it later on

**Comment by ekhaled** [comment]
Hmm, I was mistakenly thinking about this https://github.com/sveltejs/svelte/issues/893

Seems like throwing an error is the dev mode behaviour. 

Where did you get the `_updatingReadonlyProperty` from? Is that something internal to svelte?

**Comment by aphitiel** [comment]
Thank for looking into it. Yup, it's [here](https://github.com/sveltejs/svelte/blob/46c9ad4221635f8854120836863fd60fbecc0958/src/generators/dom/index.ts#L362)

**Comment by ekhaled** [comment]
@aphitiel just had a chat with @Rich-Harris and he says what you suggest won't break anything. So feel free to open a PR.

Or just let me know if you are busy and I will sort it out. 

Thanks for the investigative debugging session 😀

**Comment by ekhaled** [comment]
closed in #7
